README.md

```diff
@@ -22,17 +33,16 @@ It uses [WireGuard](https://www.wireguard.com/) to tunnel traffic between hosts.
 
 ## (Potential) Improvements
 
-* Provide Docker image for the block
-* Be smarter when TAILSCALE_KEY is not yet set in Balena
-* Provide additional configuration options
-    * subnet routing
-    * ...
-* Expose some tags in Tailscale?
-* Expose some tags in Balena?
-* Support kernel networking (instead of just userspace; also see [hslatman/tailscale-balena-rpi](https://github.com/hslatman/tailscale-balena-rpi))
-* Some easy way for checking that Tailscale tunnel works?
-* A way to refresh/reauth tailscaled state on command?
-* ...
+[x] Provide Docker image for the block
+[ ] Be smarter when TAILSCALE_KEY is not yet set in Balena
+[ ] Provide additional configuration options
+    [ ] subnet routing
+    [ ] ...
+[ ] Expose some tags in Tailscale?
+[ ] Expose some tags in Balena?
+[ ] Support kernel networking (instead of just userspace; also see [hslatman/tailscale-balena-rpi](https://github.com/hslatman/tailscale-balena-rpi))
+[ ] Some easy way for checking that Tailscale tunnel works?
+[ ] A way to refresh/reauth tailscaled state on command?
```

Comment:
```suggestion
- [x] Provide Docker image for the block
- [ ] Be smarter when TAILSCALE_KEY is not yet set in Balena
- [ ] Provide additional configuration options
    - [ ] subnet routing
    - [ ] ...
- [ ] Expose some tags in Tailscale?
- [ ] Expose some tags in Balena?
- [ ] Support kernel networking (instead of just userspace; also see [hslatman/tailscale-balena-rpi](https://github.com/hslatman/tailscale-balena-rpi))
- [ ] Some easy way for checking that Tailscale tunnel works?
- [ ] A way to refresh/reauth tailscaled state on command?
```